k8s/base/cert-issuer.yml

```diff
@@ -0,0 +1,17 @@
+apiVersion: cert-manager.io/v1
```

Comment:
In you commit message a link seems missing. (`jitsi[1]` and nothing related to `1`)